content/applications/finance/expenses.rst

```diff
@@ -365,20 +409,27 @@ in the :guilabel:`Reason to refuse Expense` field, then click :guilabel:`Refuse`
    :align: center
    :alt: Send messages in the chatter.
 
-Team managers can easily view all the expense reports for their team members. While in the
-:guilabel:`Reports to Approve` view, click on :guilabel:`Filters`, then click :guilabel:`My Team`.
-This presents all the reports for the manager's team.
+Team managers can easily view all the expense reports to approve for their team members. While in
+the :guilabel:`Reports to Approve` view, click on :guilabel:`Filters`, then click :guilabel:`My
+Team`. This presents all the reports to approve for the manager's team. Manager's can easily view
+*all* reports for their team. Simply click the X to remove the :guilabel:`To Approve` filter,
+leaving only the :guilabel:`My Team` filter active.
 
 .. image:: expenses/my-team-filter.png
    :align: center
    :alt: Select the My Team filter.
 
 .. note::
    If more information is needed, such as a receipt is missing, communication is easy from the
-   chatter. In an individual report, simply type in a message, tagging the proper person (if
-   needed), and post it to the chatter by clicking :guilabel:`Send`. The message is posted in the
-   chatter, and the person tagged will be notified via email of the message, as well as anyone
-   following.
+   chatter. In an individual report, simply click :guilabel:`Send Message`, then type in a message
+   in the text box that appears, tagging the proper person (if needed), and post it to the chatter
+   by clicking :guilabel:`Send`. The message is posted in the chatter, and the person tagged will be
+   notified via email of the message, as well as anyone following.
+
+   Note that only followers can be tagged in a message. To tag someone who is not following, add
+   them as a follower first. Click the :guilabel:`👤 (person)` icon, then click :guilabel:`Add
+   Followers`, then enter their name in the :guilabel:`Recipients` field, and finally, click
+   :guilabel:`Add Followers`.
```

Comment:
This whole section reads more as a **tip**, actually.
Also would start it like this:
`If more information is needed, such as a missing receipt, communication is  easy from the`